src/game/Objects/Unit.cpp

```diff
@@ -10202,7 +10202,7 @@ bool Unit::GetRandomAttackPoint(Unit const* attacker, float &x, float &y, float
     float angle = GetAngle(attacker);
     float sizeFactor = GetObjectBoundingRadius() + attacker->GetObjectBoundingRadius();

```

Comment:
Did you forget to change this to Combat Reach or is it supposed to be Bounding Radius?

Reply:
@balakethelock Can you reply please so we can merge this? Is this an oversight or is it intended?

Reply:
Oops it was pending this whole time :')
![image](https://github.com/vmangos/core/assets/111737968/0fd40d0a-8417-4a71-9fcc-a31929d48b2f)